pages/Monster-Jerky.md

```diff
@@ -0,0 +1,3 @@
+*"No longer hungry"*
```

Comment:
There is no need to repeat the lore on here, it doesn't really give much information about the Item.